tts-python-advanced-customization-with-ssml.ipynb

```diff
@@ -167,28 +200,32 @@
     "    1. Envelope raw text in '<speak>' tags as is required for SSML\n",
     "    2. Add '<prosody>' tag with 'pitch' attribute set to '2.5'\n",
     "    3. Add '<prosody>' tag with 'rate' attribute set to 'high'\n",
+    "    4. Add '<volume>' tag with 'volume' attribute set to '+1dB'\n",
     "\"\"\"\n",
     "raw_text = \"Today is a sunny day. But it might rain tomorrow.\"\n",
-    "ssml_text = \"\"\"<speak><prosody pitch='2.5'>Today is a sunny day</prosody>. <prosody rate='high'>But it might rain tomorrow.</prosody></speak>\"\"\"\n",
+    "ssml_text = \"\"\"<speak><prosody pitch='2.5'>Today is a sunny day</prosody>. <prosody rate='high' volume='+1dB'>But it might rain tomorrow.</prosody></speak>\"\"\"\n",
     "print(\"Raw Text: \", raw_text)\n",
     "print(\"SSML Text: \", ssml_text)\n",
     "\n",
     "\n",
-    "sample_rate_hz = 44100\n",
+    "req[\"text\"] = ssml_text\n",
     "# Request to Riva TTS to synthesize audio\n",
-    "resp = riva_tts.synthesize(\n",
-    "    text=ssml_text,\n",
-    "    voice_name=\"English-US.Female-1\",\n",
-    "    language_code=\"en-US\",\n",
-    "    encoding=riva.client.AudioEncoding.LINEAR_PCM,\n",
-    "    sample_rate_hz=sample_rate_hz,\n",
-    ")\n",
+    "resp = riva_tts.synthesize(**req)\n",
     "\n",
     "# Playing the generated audio from Riva TTS request\n",
     "audio_samples = np.frombuffer(resp.audio, dtype=np.int16)\n",
     "ipd.display(ipd.Audio(audio_samples, rate=sample_rate_hz))"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Expected results if you run the notebook:\n",
```

Comment:
Let's add a note here. If you don't see audio, please go to the docs and link the riva docs, or open it in github dev

Reply:
added a note for the first sample.